services/datamanager/datasync/sync.go

```diff
@@ -41,72 +37,34 @@ type Manager interface {
 // syncer is responsible for uploading files in captureDir to the cloud.
 type syncer struct {
```

Comment:
(putting here because I can't comment on files that were not modified it seems): Looks like `NewConnection` in datasync/client.go is now unused and can be removed